### DISCUSSION TITLE
Question about license compliance

### DISCUSSION THREAD

**Discussion by edmistond** [deepgram/community#243] [General help] [answered]
Hi all,

I'm starting to prototype an app using the Deepgram API and I want to make sure that my use case is compliant with your licensing.

Specifically, I'm working to learn dotnet MAUI and since I am deaf and this would benefit me a lot, I'd like to use the Nova API to build myself a personal captioning app using the live-streaming and speaker diarization features; most of the tools I use don't have speaker ID. Teams does, but I find the captioning quality sub-par at times and intensely dislike the way it displays captions.

Since I'm looking at this as a learning project and personal utility, I don't have plans to productize or sell this personally. I am hoping to open-source it for other deaf techies to take advantage of, with the recommendation that if they want to use it they should get their own Deepgram API key.

Is there anything in this, as described, that is potentially problematic from a license standpoint? From reading through your terms and privacy policy, I don't think there is but given that I would probably be recommending "get your own key and build it from source," I wanted to check that it's not a problem as far as Deepgram is concerned.

Thanks!

**Comment by lukeocodes** [comment, marked as answer]
We welcome and support you producing this open source project.

If you have any questions through your build, or you need any support, please don't hesitate to reach out to us. You should get a decent amount of hours from our free credit right now, but if it runs low through your development process, please let us know. We may be able to help.

Our .NET SDK is currently undergoing a few updates, so check out the open PRs there.
